pom.xml

```diff
@@ -43,6 +44,7 @@
 	        <dependency>
 	            <artifactId>mule-metadata-model-json</artifactId>
 	            <groupId>org.mule.runtime</groupId>
+				<version>1.3.0</version>
```

Comment:
This version should most likely be aligned with the version of the `mule-api`. If not incompatible with the minMuleVersion 4.3.x, then we should keep them aligned.